app/controllers/users/two_factor_authentication_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Users::TwoFactorAuthenticationController < ApplicationController
+  before_action :validate_can_manage_team, only: %i[edit destroy]
+
+  def edit
+    @user = User.find(params.fetch(:id))
+  end
+
+  def destroy
+    @user = User.find(params.fetch(:id))
+
+    @user.reset_2fa!
+
+    redirect_path = if current_organisation&.super_admin?
```

Comment:
`current_user.super_admin?`

Reply:
This is extracted from existing code, but will update it anyway.